### ISSUE TITLE
[Bug]:Fine-tuned Model Exhibits Worse Predictions for Force and Stress

### ISSUE THREAD

**Issue by GHLau** [materialsvirtuallab/matgl#453]
### Email (Optional)

318680447@qq.com

### Version

v1.1.3

### Which OS(es) are you using?

- [ ] MacOS
- [ ] Windows
- [X] Linux

### What happened?

When I finished my training process, the fine-tuned model predict the force  worse than universal model. I have no idea how to fix this issue.
[job.6184306.txt](https://github.com/user-attachments/files/17994046/job.6184306.txt)
[run_finetune.txt](https://github.com/user-attachments/files/17994052/run_finetune.txt)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by kenko911** [comment]
Hi @GHLau,
I would be very surprised if fine-tuning deep learning models can be completed within 5 epochs for a small dataset with a reasonably large batch size. Please perform training for at least a few hundred epochs. Additionally, please note that this issue is not related to bugs in our implementation, so use labels more appropriately. Thanks!

**Comment by GHLau** [comment]
> Hi @GHLau, I would be very surprised if fine-tuning deep learning models can be completed within 5 epochs for a small dataset with a reasonably large batch size. Please perform training for at least a few hundred epochs. Additionally, please note that this issue is not related to bugs in our implementation, so use labels more appropriately. Thanks!

Sorry for any confusion. Actually, after training for 200 epochs, the force predictions still have this issue. Could you please help me to check if there are any issues with my fine-tuned script?
